ts/models/groupUpdate.ts

```diff
@@ -204,9 +207,9 @@ export function getPromotedGroupUpdateChangeStr(
 export function getGroupNameChangeStr(newName: string | undefined): LocalizerComponentPropsObject {
   return newName
     ? { token: 'groupNameNew', args: { group_name: newName } }
-    : { token: 'groupNameUpdated' };
+    : { token: 'groupNameUpdated', args: undefined };
 }
 
 export function getGroupDisplayPictureChangeStr(): LocalizerComponentPropsObject {
-  return { token: 'groupDisplayPictureUpdated' };
+  return { token: 'groupDisplayPictureUpdated', args: undefined };
```

Comment:
Interesting that you have to define `args` even if undefined. I guess to keep TS happy?

Reply:
yeah. I could fix that up probably. But might be good to merge it without it when we can